features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Comments/CommentAdd.jsx

```diff
@@ -131,14 +131,16 @@ class CommentAdd extends React.Component {
             Api.addComment(apiId, comment, replyTo)
                 .then((newComment) => {
                     this.setState({ content: '' });
-                    const addedComment = newComment.body;
+                    // const addedComment = newComment.body;
                     if (replyTo === null) {
-                        allComments.push(addedComment);
-                    } else {
-                        const index = allComments.findIndex(this.filterCommentToAddReply)
-                        allComments[index].replies.list.push(addedComment);
+                        if (addComment) {
+                            addComment(newComment.body);
+                        }
+                    } else if (addReply) {
+                        addReply(newComment.body);
+                        // const index = allComments.findIndex(this.filterCommentToAddReply)
```

Comment:
Shall we remove this commented code ?

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/10226/commits/a91cfdc3ca759db39fe85d050fd17d436ae09de5